### DISCUSSION TITLE
EE-PMP%40UW/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#291] [Announcements]
# EE-PMP%40UW/

None

https://opencs.app/EE-PMP%40UW/

**Comment by XXXJumpingFrogXXX** [comment]
建议调整难度。24fall感觉人手一个了，知道几个rk1都申这个了。bar应该是有涨的

**Reply by Perrkkey** [reply]
没涨特别多，85分以上还是稳的

**Reply by chkhu** [reply]
> 没涨特别多，85分以上还是稳的

985 cs本 87均分，rej了😇

**Reply by Williamdu29** [reply]
有没有可能你是cs的，oq了

**Comment by bokehforu** [comment]
感觉没有涨得很夸张, 朋友中外合办3.48录取了, round2

**Comment by Scottgzy** [comment]
这项目居然被分到B-，不太合理。而且从找工角度来说，这项目对找工的友好性应该是A

**Comment by chkhu** [comment]
建议调整难度。看见几个dp感觉确实不止B+

**Reply by chkhu** [reply]
25fall来更新了，这项目抽风了，建议谨慎尝试

**Reply by jillhan1219** [reply]
薛定谔的bar，同时看到很好和相对弱bg的ad😂

**Comment by Zing-XII** [comment]
感觉25fall难度差不多A-了 lol

**Reply by jimlue530** [reply]
差不多在B+~A-間，跟CMU MSE差不多吧

**Comment by Skymore** [comment]
在前三个学期美国上课情况下，后面五个学期可以做五个coop😂去实习。最后一学期的最后一天也可以实习。

**Comment by tyalbakyc** [comment]
Graduate degree name 确认改名MS ECE了 https://advisingblog.ece.uw.edu/2026/04/22/uw-ee-degree-renamed-to-ece/
建议加daytime这个宝藏项目，做capstone加上3门课可以满足ECE 24学分要求，其他全修CS, AMATH, INFO软课, 如果上两年可以修到7门以上的系外(CS, AMATH, INFO)课，daytime本身也有软课，还可以将一门非ECE的课投过petition转成系内学分。

**Reply by tyalbakyc** [reply]
Daytime难度：24-25年acceptance rate在16
发了129个offer https://www.ece.uw.edu/wp-content/uploads/2026/04/Masters-Flyer-Full-Apr-2026-v1.pdf
